mentoring/programs/office-hours.md

```diff
@@ -0,0 +1,35 @@
+title: Office Hours/ Mentoring
+weight: 4
+description: Office hours of different SIGs
+---
+
+* API Machinery - 
+    * [Kubebuilder and Controller Runtime Meeting - Second Wednesday every month at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1Ih-2cgg1bUrLwLVTB9tADlPcVdgnuMNBGbUl4D-0TIk/edit?usp=sharing)
+* Architecture - 
+    * [Enhancements Subproject Meeting - Biweekly Tuesday at 08:30 PT (Pacific Time)](https://bit.ly/k8s-enhancements-agenda)
+    * [Production Readiness Office Hours - Biweekly Wednesday at 12:00 PT (Pacific Time)](https://docs.google.com/document/d/10QkXwiZfBL7wBKHHFslooXvGDKHA75jkkgiIXuKFryM/edit?usp=sharing)
+    * [Code organization Office Hours - Biweekly Thursday at 14:00 PT (Pacific Time)](https://docs.google.com/document/d/1HtTI0rJEGP_MSf6eO87aCmx_tzpovPAAg7U2Zxwm8FE/edit#)
+    * [Conformance office Hours - Biweekly Tuesday at 19:00 UTC](https://docs.google.com/document/d/1W31nXh9RYAb_VaYkwuPLd1hFxuRX3iU0DmaQ4lkCsX8/edit#)
+* Cluster Lifecycle - 
+    * [Cluster API (AWS implementation) office hours - Biweekly Monday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1ojR0jooQaHEuz3gGW6DxcZlsws9ZX7jHPC8walm1G3A/edit)
+    * [Cluster API office hours - Biweekly Wednesday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1fQNlqsDkvEggWFi51GVxOglL2P1Bvo2JhZlMhm2d-Co/edit)
+    * [Cluster Addons meeting - Biweekly Tuesday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/10_tl_SXcFGb-2109QpcFVrdrfnVEuQ05MBrXtasB0vk/edit)
+    * [Image Builder office hours - Biweekly Wednesday at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1YIOD0Nnid_0h6rKlDxcbfJaoIRNO6mQd9Or5vKRNxaU/edit)
+    * [Kubespray Office Hours - Biweekly Wednesday at 08:00 PT (Pacific Time)](https://docs.google.com/document/d/1oDI1rTwla393k6nEMkqz0RU9rUl3J1hov0kQfNcl-4o/edit)
+    * etcdadm Office Hours - Biweekly Monday at 09:00 PT (Pacific Time)
```

Comment:
```suggestion
    * [etcdadm Office Hours - Biweekly Monday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/1b_J0oBvi9lL0gsPgTOrCw1Zlx3e7BYEuXnB3d2S15pA/edit)
```
I was able to find their meeting notes link from their slack channel. If you'd like to do another PR, sigs.yaml could be updated to include the link to the doc:
https://docs.google.com/document/d/1b_J0oBvi9lL0gsPgTOrCw1Zlx3e7BYEuXnB3d2S15pA/edit

Similar to your other one with updating the timezones :)

Reply:
Sure thing will do this

Reply:
@mrbobbytables https://github.com/kubernetes/community/pull/4986 here you go :)